docs/en/stack/security/authorization/built-in-roles.asciidoc

```diff
@@ -39,11 +39,13 @@ suitable for writing beats output to {es}.
 
 [[built-in-roles-data-frame-transforms-admin]] `data_frame_transforms_admin` ::
 Grants `manage_data_frame_transforms` cluster privileges, which enable you to
-manage data frames.
+manage data frame transforms. This role also includes the `reserved_ml` {kib}
```

Comment:
@kobelb will this level of Kibana privilege be documented anywhere?  It doesn't seem we go into that level of detail in  https://www.elastic.co/guide/en/kibana/master/kibana-privileges.html so maybe we should omit it here too?  e.g. Change it to something like "Users with this role have all {ml} feature privileges in Kibana".

Reply:
That's a good point... we've generally hidden the concept of "Kibana reserved privileges" from end-users, so the phrasing which you've recommended might make this less confusing to users.

Reply:
Thanks @kobelb !  @droberts195 I've drafted suggestions for the changes.